ui/shared/components/panel/variants/RnaSeqTpm.jsx

```diff
@@ -38,39 +38,40 @@ const PLOT_OPTIONS = {
   yAxisUnit: 'TPM',
 }
 
-const launchGtex = (geneId, tpms) => (gencodeId) => {
-  const tissues = [...new Set(Object.values(tpms).map(({ sampleTissueType }) => sampleTissueType))]
-  const boxplotData = Object.entries(tpms).map(([individual, { tpm }]) => ({ label: individual, data: [tpm] }))
+const launchGtex = (geneId, familyGuid) => (gencodeId) => {
```

Comment:
the control flow changes here - before we already had the sample specific data so we formatted that data and fetched the gtex and rdg aggregate data. Now, we first do one request to fetch the sample and rdg aggregate data, and then do a followup request to gtex for the tissues that are returned from the API